core/lib/src/catcher.rs

```diff
@@ -153,7 +153,7 @@ macro_rules! default_catchers {
         let mut map = HashMap::new();
 
         $(
-            fn $fn_name<'r>(req: &'r Request<'_>) -> futures_util::future::BoxFuture<'r, response::Result<'r>> {
+            fn $fn_name<'r>(req: &'r Request<'_>) -> futures::future::BoxFuture<'r, response::Result<'r>> {
```

Comment:
Should this be `$crate::futures::`, to be safe?

Reply:
Since this macro isn't usable in any other module(s), the only conflict would be with another valid path `futures::future::BoxFuture` reachable from `mod defaults` below. IMO `$crate` will look verbose/out of place here with no benefit.